front/app/components/admin/SortableTableHeader/index.tsx

```diff
@@ -11,6 +11,7 @@ const SortableLink = styled.a`
 interface Props {
   direction?: 'ascending' | 'descending' | null;
   onToggle: () => void;
+  children: React.ReactNode;
```

Comment:
Hi Luuc!
This should not be necessary, type `FC` already includes children - please pull the latest master before fixing lint errors :)

Reply:
Hey, I just pulled in the latest version of master and removed this type again, but now I get the same error as before:
![image](https://user-images.githubusercontent.com/13822005/132013835-cdca9f9a-bbbb-4fdd-a4b3-968a48fd8db6.png)

Reply:
This is strange, let me look into it :)

Reply:
Ok, there seems to be an issue with this particular rule and using `React.FC`. I think it's perfectly fine to disable it, since we are relying on typescript from prop type validation anyway! So to disable, just switch it off in the eslint config :)